src/hotspot/share/gc/parallel/psCardTable.cpp

```diff
@@ -389,20 +389,7 @@ void PSCardTable::verify_all_young_refs_imprecise() {
 
 bool PSCardTable::addr_is_marked_imprecise(void *addr) {
```

Comment:
Does it make sense to keep this name when we don't have the "precise" counterpart and what all it does is checking if the card is dirty?

Reply:
Indeed odd. I will rename it to sth else in a follow-up PR. Even "marked" is weird here.

Reply:
Sounds good, my first thought was `addr_is_dirty(void* addr)`.